hw_06/task_02.js

```diff
@@ -0,0 +1,100 @@
+// 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+
+
+const competitorPizzas = ['Pepperoni', 'Capricciosa', 'Diavola', '4 formaggi', 'Hawai'];
+
+function comparePizzas (arrayPizzas) {
+    
+    const comparasionResult = [];
+
+    for (let i = 0; i < arrayPizzas.length; i++) {
+           if (!competitorPizzas.includes(arrayPizzas[i])) {
+               comparasionResult.push(arrayPizzas[i]);
+        }
+    }
+
+    if (comparasionResult.length === 0) {
+        return null;
+    }
+
+    return comparasionResult;
+    
+}
+
+console.log(comparePizzas(['Pepperoni', 'Siciliana', '4 stagioni', 'Romana', 'Diavola']))
+
+  
+
+
+// 2. Написать функцию, которая принимает предложение (слова разделенные только пробелами) в качестве параметра и выводит в консоль слово с наибольшим количеством букв. 
+//   Если таких слов несколько - выводит их все.
+
+
+function hasMoreLetters (sentence) {
+    sentenceIsArray = sentence.replace('.','').split(' ');
+    
+    let wordLenght = 0;
+    const longestWord = [];
```

Comment:
`sentenceIsArray` - to `let` or to `const`, that is the question :)

Related to the next suggestion
```js
const longestWord = [];
let longestWord = [];
```